### DISCUSSION TITLE
RTP live stream

### DISCUSSION THREAD

**Discussion by MahdiMotallebi** [katspaugh/wavesurfer.js#3919] [Q&A]
Hello,
I get RTP live stream from Raspberry Pi  with this API "http://localhost:5000/API/voice/stream"
how can i handle live stream?to play and visualization

**Comment by katspaugh** [comment]
Streams aren’t supported. It’s mentioned in the readme (see the FAQ section).

**Reply by MahdiMotallebi** [reply]
But how work Mic Plugin?, can i get stream and convert it to audioBuffer then pass to renderStream function that there is in Mic Plugin?

**Reply by katspaugh** [reply]
That theoretically would work. It wouldn’t be a buffer but a MediaElementSourceNode (like here: https://wavesurfer.xyz/examples/?webaudio.js).
